fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -37,6 +37,9 @@
     private static final Logger LOGGER = LoggerFactory
             .getLogger(FedoraUserSecurityContext.class);
 
+    // Defined here because FedoraJcrTypes is in fcrepo-http-api, which would create a cyclic dependency
+    protected static final String JCR_CONTENT = "jcr:content";
```

Comment:
Thanks for the PR.
Would you agree that your PR creates a dependency from fcrepo-auth-common on to fcrepo-http-api? This new dependency is coming transitively from some other artifact... but exists, nonetheless. If you explicitly add the dependency into fcrepo-auth-common's pom.xml, Maven complains.

Are you comfortable relying on this transitive coupling?
Am I missing something?